src/raftstore/store/snap.rs

```diff
@@ -1082,17 +1095,29 @@ pub struct SnapManager {
     // directory to store snapfile.
     core: Arc<RwLock<SnapManagerCore>>,
     ch: Option<SendCh<Msg>>,
+    limiter: Arc<SnapshotIOLimiter>,
 }
 
 impl SnapManager {
-    pub fn new<T: Into<String>>(path: T, ch: Option<SendCh<Msg>>) -> SnapManager {
+    pub fn new<T: Into<String>>(
+        path: T,
+        ch: Option<SendCh<Msg>>,
+        min_bytes_per_time: usize,
```

Comment:
can you tell me what these args mean fro the SnapManager? 

You should pass a SnapManager config or the limiter direclty